package.json

```diff
@@ -9,31 +9,49 @@
     "build:min": "mkdir -p build && MDL_ENV=production webpack -p",
     "dist": "npm run clean && npm run build && npm run build:min",
     "dev": "npm run clean && MDL_ENV=development webpack-dev-server --content-base demos --inline --hot",
-    "lint:js": "eslint --fix packages webpack.config.js",
-    "postinstall": "lerna bootstrap"
+    "lint:js": "eslint --fix packages test webpack.config.js karma.conf.js",
```

Comment:
This one's meant to be manually used by developers, not as part of some build process, right?